packages/slice-machine/components/ChangesItems/ChangesItems.tsx

```diff
@@ -33,6 +33,10 @@ export const ChangesItems: React.FC<ChangesItemsProps> = ({
 
   const { sliceFilterFn, defaultVariationSelector } = modalPayload;
 
+  const screenshotChangesSlices = unSyncedSlices.filter(
```

Comment:
this is a fix for an error I encountered during coding: user were able to access screenshot modal of a deleted slice, and therefore could not add a screenshot. This removes deleted slices from the list of slices to display